contrib/storage-kafka/src/main/java/org/apache/drill/exec/store/kafka/KafkaGroupScan.java

```diff
@@ -227,6 +229,33 @@ private void init() {
     }
   }
 
+
+  /** Workaround for Kafka > 2.0 version due to <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-505%3A+Add+new+public+method+to+only+update+assignment+metadata+in+consumer">KIP-505</a>.
```

Comment:
```suggestion
  /**
  Workaround for Kafka > 2.0 version due to KIP-505. It can be replaced with Kafka implementation once it will be introduced.
```

Reply:
Why do you prefer to remove the direct link? I think it's a very convenient thing and it's fast to check whether it was resolved.

Reply:
@rymarm Thanks for the question.
In general, we can record any information of limitations into the README.md of plugins, such as `contrib/format-xml/README.md`. Then, we can keep the comment as brief as possible, and here we focus on providing an alternative function. So the KIP-505 keyword is enough.

Reply:
@luocooong Thank you for explaining! Followed your suggestion.